README.md

```diff
@@ -43,7 +43,7 @@ Example configuration file:
 LOG_LEVEL_MAIN = INFO
 RUN_FLAKE8 = False
 RUN_ANSIBLE_TEST = False
-INFRA_PULP = False
+INFRA_LOCAL_IMAGE = False
```

Comment:
Add `INFRA_LOCAL_DOCKER = False`

Below the example config file, please add something like the following to explain the `ansible-test` options:

* RUN_ANSIBLE_TEST - Set to `True` to run `ansible-test` during collection import. If `True`, requires Ansible to be installed, unless container image execution is enabled.
* INFRA_LOCAL_IMAGE - Set to `True` to run `ansible-test` sandboxed within a container image. Defaults to  False. Requires installation of either Podman or Docker to run the container.
* INFRA_LOCAL_DOCKER - Set to `True` to run the `ansible-test` container image via Docker; otherwise, Podman will be used.

Reply:
I thought we were just going to pick one local execution, docker or podman, if so i vote docker.

Reply:
What's wrong with offering both? Plus, Podman doesn't require a daemon.

Reply:
sounds good